stubs/setuptools/METADATA.toml

```diff
@@ -1,5 +1,13 @@
 version = "67.4.*"
-requires = ["types-docutils"]
+extra_description = """\
+Having [setuptools](https://pypi.org/project/setuptools/) installed results in incorrect \
+[distutils](https://docs.python.org/3/distutils/introduction.html) typing. \
+`types-setuptools` makes no attempt at correcting this issue. As of \
+`setuptools>=60.0.0`, you [should not be using `distutils` directly](\
+https://setuptools.pypa.io/en/latest/deprecated/distutils-legacy.html). \
+`distutils` is deprecated as of Python 3.10 and will be fully removed in Python 3.12. \
+(see [PEP 632](https://peps.python.org/pep-0632/))
+"""
```

Comment:
I'm not sure that this is necessary -- it sort of feels like it's `setuptools`'s problem rather than ours. If we _do_ want some kind of a note here, I'd prefer it if it were slightly more concise.

Reply:
Does this look better to you?
> Having [setuptools](https://pypi.org/project/setuptools/) installed results in incorrect [distutils](https://docs.python.org/3/distutils/introduction.html) typing. `types-setuptools` makes no attempt at correcting this issue. See [Porting from Distutils](https://setuptools.pypa.io/en/latest/deprecated/distutils-legacy.html) and [PEP 632 – Deprecate distutils module](https://peps.python.org/pep-0632/).

Reply:
Hmm, the more I think about it, the more I lean towards not including this note at all. It feels like unnecessary finger-pointing at `setuptools` from us. The `distutils` situation is very frustrating for me, a typeshed maintainer, but as far as I know we haven't received any complaints from users of our stubs about incorrect `distutils` typing, so it sort of feels like this note is attempting to solve something that may not _actually_ be much of an issue for users. If we do get complaints or queries from users, we can maybe rethink this, but for now I think we can _probably_ do without this :)

Reply:
I can see what you mean. I also expect the amount of users who'll actually land on the PyPI page to be taught about the issue of mixing setuptools and distutils together to be really small. (pyright ships stubs, and the average mypy user probably installs stubs from mypy's CLI suggestion).